pkg/dashboard/templates/email.gohtml

```diff
@@ -0,0 +1,26 @@
+{{ define "email" }}
+<div class="email-box" id="email-box">
+  <details open>
+    <summary class="--hideMarker">
+      <h2 class="email-box__toggle-title">
+        Add cost estimates to your Goldilocks dashboard!
+        <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+        ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+      </h2>
+    </summary>
+
+    <section class="email-box__content">
+      <p>Enter your email to get a free estimate of the cost impact of applying Goldilocks’ recommendations.</p>
+      <input id="email-box__email-input" class="email-box__email-input" type="email" placeholder="Email" />
+      <div>
+        <label class="email-box__checkbox-container">
+          <input type="checkbox" id="emailbox__checkbox">
+          <span class="email-box__checkmark"></span>
+          <span class="email-box__checkbox-content">By entering your email address, you acknowledge that we may use it as our Privacy Policy describes.</span>
```

Comment:
I made the privacy policy text as a link. However, I don't know how to put it outside the `label`. If you have any idea, please feel free to let me know. Thank you

Reply:
Thanks for updating this. I do have an idea for moving the link outside the `label`, but I'll add it in a new review comment, since this one is marked as "Outdated."